.pre-commit-hooks.yaml

```diff
@@ -2,11 +2,21 @@
 
 - id: terraform-fmt
   name: Terraform fmt
-  description: Rewrites all Terraform configuration files to a canonical format.
+  description: Rewrites all Terraform configuration files to a canonical format
   entry: hooks/terraform-fmt.sh
   language: script
   files: \.tf$
   exclude: \.+.terraform\/.*$
+  require_serial: true
+
+- id: terraform-validate
+  name: Terraform validate
+  description: Validates all Terraform configuration files
+  entry: hooks/terraform-validate.sh
+  language: script
+  files: \.tf$
+  exclude: \.+.terraform\/.*$
+  require_serial: true
```

Comment:
Looks like this is a feature for `pre-commit` > 1.13 (same with the parallelization). Internally at gruntwork we use `pre-commit` 1.11 because circleci has issues installing anything greater than that.

Nothing for you to change: this is a reminder to myself to mention in the release notes that we require `pre-commit` > 1.13 now and we need to test for compatibility.